bin/matrix-publish.js

```diff
@@ -120,77 +127,103 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
     }
   });
   
+  var localReadmeFileName = 'DEVELOPER.MD';
+  var remoteReadmeFileName = 'readme.md';
```

Comment:
lets make the local name `README.MD`. Developer is more for a dev readme. Lets remember to change the baseapp.tar